dist/container.js

```diff
@@ -1990,29 +2000,44 @@
 			// Do some validation on the style settings
 			if (true)
 			{
-				if (!styles.color || !/^(black|white|red|yellow|pink|blue)(-semi)?$/.test(styles.color))
+				if (!styles.color ||
+					!/^(black|white|red|yellow|pink|blue)(-semi)?$/.test(styles.color)
```

Comment:
Jut a nit picky thing, but can we move the regex into a var, and then get the if statement on one line?